docs/building/windows-core.md

```diff
@@ -44,10 +44,7 @@ Invoke-WebRequest -Uri https://raw.githubusercontent.com/dotnet/cli/rel/1.0.0/sc
 If you have any problems installing `dotnet`,
 please see their [documentation][cli-docs].
 
-If you are using Windows 7, Windows Server 2008 or Windows Server 2012
-you will also need to install
-[Visual C++ Redistributable for Visual Studio 2012 Update 4][redist-2012]
-and [Visual C++ Redistributable for Visual Studio 2015][redist-2015].
+You will also need to install a recent version of Visual Studio with the optional feature 'Common Tools for Visual C++' installed. The free [Community edition of Visual Studio 2015](https://beta.visualstudio.com/visual-studio-community-vs/) should work fine.
```

Comment:
I believe those requirements are reflected in this PR @mirichmo. Was there a particular detail you see missing? I'm happy to tweak this to make it more clear.